types/node/index.d.ts

```diff
@@ -2342,7 +2342,19 @@ declare module "url" {
         query?: string | null | ParsedUrlQuery;
```

Comment:
Maybe remove this, and make the `boolean` `parse` overload return `UrlParsedQuery | UrlStringQuery`?

Reply:
seems `Url.url` is used at several other modules so I will keep it.